Makefile

```diff
@@ -0,0 +1,13 @@
+# `make build` builds the service images. Docker and goreleaser must be
```

Comment:
I'm not convinced we need a Makefile to execute shell scripts. It would be better to just execute the shell scripts directly.

That being said, I'm also not sure we should be using shell scripts either. What are we trying to do here?

Reply:
Was thinking we might need to group scripts in a single command, it is probably not needed now. Will update, thanks!